src/components/MapView/responder/index.ts

```diff
@@ -1,8 +1,3 @@
-import {PanResponder} from 'react-native';
+import SwipeInterceptPanResponder from '../../SwipeInterceptPanResponder';
 
-const responder = PanResponder.create({
-    onMoveShouldSetPanResponder: () => true,
-    onPanResponderTerminationRequest: () => false,
-});
-
-export default responder;
+export default SwipeInterceptPanResponder;
```

Comment:
The same things can be applied to `MapView/responder/index.android.ts`.

Reply:
The responder in `MapView/responder/index.android.ts` uses `onStartShouldSetPanResponder`, not `onMoveShouldSetPanResponder`, so that is different.